Fix charter.md

agiudiceandrea opened the pull request:
Add missing chapters' numbers, fix incorrect chapters's level, fix outdated link.

For reference: https://web.archive.org/web/20240527021651/https://www.qgis.org/en/site/getinvolved/governance/charter/index.html

yjacolin commented on source/site/forusers/alldownloads.rst:
```diff
@@ -458,3 +458,7 @@ QGIS Testing warning
+Installing from Source
+======================
+
+To build and install QGIS from source see https://github.com/qgis/QGIS/blob/master/INSTALL
```
Could you use this link instead: http://htmlpreview.github.io/?https://raw.github.com/qgis/QGIS/master/doc/INSTALL.html

andrewharvey replied:
sure, I've changed the link

yjacolin replied:
thanks it could be even better if you can use something like 
```
[QGIS from source](http://htmlpreview.github.io/?https://raw.github.com/qgis/QGIS/master/doc/INSTALL.html).
```
Which will be displayed like this:
[QGIS from source](http://htmlpreview.github.io/?https://raw.github.com/qgis/QGIS/master/doc/INSTALL.html).
Sorry to forget this in my previous comment.

andrewharvey replied:
I've used similar wording as the Sources tab at https://www.qgis.org/en/site/forusers/download.html. For some reason I never found that tab and only found the guide at https://www.qgis.org/en/site/forusers/alldownloads.html.

yjacolin commented:
@andrewharvey sorry I was wrong about the tags element. I updated your branch directly. I will merge it after the test. Thanks a lot!

agiudiceandrea committed:
Fix charter.md

Add missing chapters' numbers, fix incorrect chapters's levels, fix outdated link

Xpirix reviewed (approved):
LGTM, thanks @agiudiceandrea
